### ISSUE TITLE
Edit File name and File description are not keyboard navigable. 

### ISSUE THREAD

**Issue by Hanal-code** [techmatters/terraso-web-client#392]
## Description
Keyboard users cannot edit the file name and file description - a11y issue

## Steps To Reproduce
Steps to reproduce the behavior. Describe what you typed, clicked, tapped, swiped, saw and heard.
1. Go to URL: ['...'](https://app.staging.terraso.org/groups/academic-research-test#) OR any landscape/group that has a shared file. 
2. Navigate the page using the keyboard. 
3. Notice the file name is not focusable. The focus skips to Delete button. 

## Expected behavior
The file name and the file description are focusable.

## Actual behavior
The file name and the file description are NOT focusable.

**Comment by paulschreiber** [comment]
@Hanal-code this is on staging to verify.

**Comment by Hanal-code** [comment]
Verified
<img width="507" alt="image" src="https://user-images.githubusercontent.com/92342777/172245333-de5a54a4-5a8d-45e8-bc7d-7c6f44ad1690.png">
